cabal-install/src/Distribution/Client/ProjectConfig.hs

```diff
@@ -754,9 +754,9 @@ renderImplicitBadPackageLocation bpl = case bpl of
     BadLocGlobEmptyMatch pkglocstr ->
         "No cabal.project file or cabal file matching the default glob '"
      ++ pkglocstr ++ "' was found.\n"
-     ++ "Please create a package description file <pkgname>.cabal "
-     ++ "or a cabal.project file referencing the packages you "
-     ++ "want to build."
+     ++ "No <pkgname>.cabal was found and the current working directory is not "
+     ++ "part of a cabal project. To build (or run) local packages you need a cabal file "
```

Comment:
Not sure if a directory can be part of a cabal.project without having a .cabal file.
In my tests i am getting other errors if i list a subdirectory in the `packages`field of a cabal.project and that dorectory has no .cabal file